### PR TITLE
[HAPI2] Initial support of starting & stopping HAPI2 plugins automatically

### PR DESCRIPTION
**Pull request by ashie**
If armPluginInfo.path isn't empty, execute it with an argument "start" or "stop" when a instance of
HatoholArmPluginGateHAPI2 starts or stops.

### CODE REVIEW

**Comment by ashie** [comment]
To tell the truth, it's not enough yet.
It doesn't provide a away to find the process to stop yet.

File: server/src/HatoholArmPluginGateHAPI2.cc [hunk 1]
```diff
@@ -321,6 +322,104 @@ struct HatoholArmPluginGateHAPI2::Impl
...
 	};
+
+	bool isPluginControlScriptAvailable(void)
+	{
+		if (m_pluginInfo.path.empty())
+			return false;
+		bool executable = g_file_test(m_pluginInfo.path.c_str(),
+					      G_FILE_TEST_IS_EXECUTABLE);
+		if (!executable)
+			MLPL_DBG("The plugin control script isn't executable:"
```
Comment (kz0817) [inline server/src/HatoholArmPluginGateHAPI2.cc]: The higher message level would be better. For example INFO or WARN.
And users get a chance to notice the cause of problem.

Comment (kz0817) [inline server/src/HatoholArmPluginGateHAPI2.cc]: > Or, is a message of failure of the execution shown in other place ?

Yes, the failure message is shown in runPluginControlScript. However, the reason isn't.
So here the message including a cause should be logged with more higher level.

Comment (ashie) [inline server/src/HatoholArmPluginGateHAPI2.cc]: > So here the message including a cause should be logged with more higher level. 

I think so too. I'll change it.

Comment (ashie) [inline server/src/HatoholArmPluginGateHAPI2.cc]: I fixed it.

**Comment by ashie** [comment]
Thanks!

**Pull request merged**